Added functionality to !bible command

futuremoe opened the pull request:
If user input is correct, the bot will fetch response from
dailyverses.net with book, chapter and verse from user input.

!help command output has been updated with correct usage

Commit:
Added functionality to !bible command
If user input is correct, the bot will fetch response from
dailyverses.net with book, chapter and verse from user input.

!help command output has been updated with correct usage

IAmNotGoodWithComputers commented on botcommand_bible.go:
```diff
@@ -1,33 +1,56 @@
+func (b *BibleCommand) getBibleVerse(context *MessageContext) {
+	msgBook := strings.Split(context.Message.Content, " ")[1]
+	msgChpt := strings.Split(strings.Split(context.Message.Content, " ")[2], ":")[0]
+	msgVrs := strings.Split(strings.Split(context.Message.Content, " ")[2], ":")[1]
```
invalid input leads to crashing the bot, you could check if indexes exist before accessing them and eventually output that the desired bible verse was not found. You can let the bot crash by typing

`!bible asdf`

tiananmensquare commented:
OK I generally like your PR a lot, just a tiny bug to be fixed and I'll merge it :) thanks again for contributing!

Commit:
False user input leads to no message.
Chapter and verse are now checked to be numbers and higher than 0.
Since dailyverses redirects when a "wrong" url is requested, the status code is checked to be 200 before the verse is outputted.

Commit:
Fixed bug where user could not enter more than two wors e.g. "2 corinthians" as book.
